src/api.ts

```diff
@@ -56,6 +56,9 @@ export class KubernetesAPI implements IKubernetesAPI {
             v1beta1: () => ({
                 customResourceDefinitions: () => this.c("/apis/apiextensions.k8s.io/v1", "/customresourcedefinitions"),
```

Comment:
Hang on, did we previously actually use `v1/customresourcedefinions` when the user wanted to use `v1beta1`? 😱